src/component/common/SaveChangesButton/SaveChangesButton.tsx

```diff
@@ -0,0 +1,16 @@
+import PermissionButton from '../PermissionButton/PermissionButton';
+
+interface ISaveChangesButtonProps {
+    permission: string;
+}
+
+export const SaveChangesButton = ({
+    permission,
+    ...rest
+}: ISaveChangesButtonProps) => {
+    return (
+        <PermissionButton permission={permission} type="submit" {...rest}>
+            Save
```

Comment:
Should this use a `{resourceName}` as well?

Reply:
> Should this use a `{resourceName}` as well?
I don't think so because we want to show only `Save` and not ~`Save x`~

![Screen Shot 2022-02-24 at 8 51 53 AM](https://user-images.githubusercontent.com/47088512/155481381-fc325ee5-5dcd-49cc-a470-b8e39caff962.png)